src/client_side.h

```diff
@@ -202,7 +202,7 @@ class ConnStateData : public Server, public HttpControlMsgSink, private Independ
     /// noteTakeServerConnectionControl() callback parameter
     class ServerConnectionContext {
     public:
-        ServerConnectionContext(const Comm::ConnectionPointer &conn, const HttpRequest::Pointer &req, const SBuf &post101Bytes): preReadServerBytes(post101Bytes), conn_(conn) { conn_->enterOrphanage(); }
+        ServerConnectionContext(const Comm::ConnectionPointer &conn, const HttpRequest::Pointer & /* req */, const SBuf &post101Bytes): preReadServerBytes(post101Bytes), conn_(conn) { conn_->enterOrphanage(); }
```

Comment:
Is there a good reason to keep this unused parameter? If not, please remove it.

Same request for similar contexts, of course.

Reply:
Removed unused parameter. It's not always easy nor possible, as in many cases it's virtual classes